.env.docker

```diff
@@ -22,3 +22,6 @@ ORIGIN=https://catfish-pro-wolf.ngrok-free.app
 
 ANDROID_SHA256HASH=47:CC:4E:EE:B9:50:59:A5:8B:E0:19:45:CA:0A:6D:59:16:F9:A9:C2:96:75:F8:F3:64:86:92:46:2B:7D:5D:5C
 ANDROID_PACKAGENAME=foundation.algorand.demo
+
+TURN_USERNAME="fc7708976bf5d60be20c5a1d"
```

Comment:
This needs to be under `./example/.env` and not a part of the service config

They must also be prefixed with `VITE_` in order to be defined in the React app